systemvm/debian/opt/cloud/bin/setup/cloud-early-config

```diff
@@ -224,11 +209,11 @@ start() {
 
   config_guest
   get_boot_params
-  get_systemvm_type
   patch
   sync
   sysctl -p
 
+  export TYPE=$(grep -Po 'type=\K[a-zA-Z]*' $CMDLINE)
```

Comment:
I do not think this is an improvemnet over the very legible function used before. Even when being less code it is not as clear and it seems less relyable to me

Reply:
this is about 10x-20x faster (based on `time` output) than the for-loop before which is why I'm using it.

Reply:
You're right @fmaximus, not sure why smoketests passed. /cc @PaulAngus looks like we'll need rc2.

Reply:
@fmaximus see https://github.com/apache/cloudstack/pull/2855